Buddies.API/Controllers/ProjectsController.cs

```diff
@@ -168,6 +174,54 @@ public async Task<ActionResult> CreateProject(CreateProjectRequest project)
             return StatusCode(StatusCodes.Status201Created, dbProject.ProjectId);
         }
 
+        /// <summary>
+        /// API route PUT /api/v1/projects/skills for updating project skills.
+        /// </summary>
+        [HttpPut("skills/{id}")]
+        [Authorize]
+        public async Task<ActionResult> UpdateProjectProfile(int id, UpdateProjectSkillRequest skills)
+        {
+            var project = _context.Projects
+                .Include(project => project.Owner)
+                .Include(project => project.Skills)
+                .Where(project => project.ProjectId == id)
+                .FirstOrDefault();
+
+            if (project == null)
+            {
+                return NotFound("PROFILE NOT FOUND");
+            }
+
+            if (project.Owner != _userManager.GetUserAsync(User).Result) { return Unauthorized("You must be the owner"); }
+            var i = 0;
+            for (i = 0; i < skills.Skills.Count; i++)
+            {
+                if (i >= project.Skills.Count)
+                {
+                    var newSkill = new ProjectSkill(skills.Skills[i].Name);
+                    project.Skills.Add(newSkill);
+                }
+                else
+                {
+                    project.Skills[i].Name = skills.Skills[i].Name;
+                }
+                if (i == 2)
```

Comment:
You could probably use `min(skills.Skills.Count, 2)` in the conditional to avoid this. Then below I would do `var j = i + 1`. Its usually not a good idea to mutate the counter of a for loop in the conditional of the loop and the body (unless you really have to).